contracts/CMakeLists.txt

```diff
@@ -6,6 +6,8 @@ add_subdirectory(test_api)
 add_subdirectory(simpledb)
```

Comment:
Please move test contracts to `tests/tests/contracts`

Reply:
Sure, will do that